opendrift/scripts/opendrift.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+
+from opendrift.models.leeway import Leeway
+from opendrift.models.openoil import OpenOil
+from opendrift.models.shipdrift import ShipDrift
+from datetime import datetime, timedelta
+import click
+import sys
+import json
+import math
+
+
+def get_leeway_model(loglevel):
+    model = Leeway(loglevel=loglevel)
+    set_config_override(model)
+    return model
+
+def get_openoil_model(loglevel):
+    model = OpenOil(location='NORWAY', loglevel=loglevel)
+    set_config_override(model)
+    return model
+
+def get_shipdrift_model(loglevel):
+    model = ShipDrift(loglevel=loglevel)
+    set_config_override(model)
+    return model
+
+
+def set_config_override(model):
+    model._config['seed:number']['default'] = 1000
+    model._config['seed:number']['value'] = 1000
+    model._config['seed:number']['max'] = 100000
+
+    model._config['general:time_step_minutes']['default'] = 15
+    model._config['general:time_step_minutes']['value'] = 15
+    model._config['general:time_step_minutes']['min'] = 5
+
+    model._config['general:time_step_output_minutes']['default'] = 60
+    model._config['general:time_step_output_minutes']['value'] = 60
+    model._config['general:time_step_output_minutes']['min'] = 30
+
+    try:
+        model._config['seed:seafloor']['level'] = 2
+        model._config['seed:z']['level'] = 2
+    except KeyError:
+        pass
+
+models = {
+    'leeway': get_leeway_model,
+    'openoil': get_openoil_model,
+    'shipdrift': get_shipdrift_model
+}
+
+
+plots = {
```

Comment:
This is fine for now, but further down the road it would be nice to generalize this functionality, so that "any" postprocessing method can be applied. That requires some careful thinking.
Then also post-processing methods could be made available through Drifty.